Aptabase.uplugin

```diff
@@ -0,0 +1,22 @@
+﻿{
+	"FileVersion": 3,
+	"VersionName": "0.1.0",
+	"EngineVersion": "5.2",
+	"FriendlyName": "Aptabase",
+	"Description": "Open Source, Privacy-First and Simple Analytics for Mobile, Desktop and Web Apps",
+	"Category": "Analytics",
+	"CreatedBy": "Sumbit Labs Limited",
+	"CreatedByURL": "https://aptabase.com/",
+	"DocsURL": "",
```

Comment:
@goenning here is the most user-facing information about the integration

and it looks like this, let me know if you think anything should be changed:

![image](https://github.com/aptabase/aptabase-unreal/assets/21221169/2e15e649-604e-4850-b875-feb8d0978702)

Reply:
Looks great :)